src/query/storages/fuse/src/operations/common/processors/sink_commit.rs

```diff
@@ -361,6 +362,12 @@ where F: SnapshotGenerator + Send + 'static
                         if let Some(files) = &self.copied_files {
                             metrics_inc_commit_copied_files(files.file_info.len() as u64);
                         }
+                        for segment in self.abort_operation.segments.iter() {
```

Comment:
What does `self.abort_operation.segments` mean?

Reply:
the storage first add all new generated segments to the `abort_operation`, if commit failed, all things in abort_operation will be deleted, include cache.